src/v3/src/components/TextWithHtml/TextWithHtml.tsx

```diff
@@ -66,8 +67,9 @@ const TextWithHtml: UISchemaElementComponent<{
           },
         })}
         onClick={handleClick}
-        dangerouslySetInnerHTML={{ __html: content }}
-      />
+      >
+        {ReactHtmlParser(content, { transform: uischema.contentTransformer })}
```

Comment:
Could `ReactHtmlParser` also be encapsulated into the transformer? If so, are there major downsides?

Reply:
I considered this (I also considered adding it to our `loc` util function), the problem with that is it alters what is returned (so we will need to type it as `string | h.jsx.Element`. But additionally, it would require us to go through each flow/scenario and determine which contains spans, anchors, etc embedded in the translation keys we use, to determine if we need to use the parser (not just the ones that use the `$1` tags). So this approach is more of a catch all.